src/context/XplorersAIContext.tsx

```
@@ -0,0 +1,98 @@
+  MouseEvent,
+} from "react";
+import { ChatMessage, FileUploadEvent, XploresAIType } from "@/interface";
+import { callAzureOpenAI } from "@/app/api/chat/ai";
+
+const XplorersAIContext = createContext<XploresAIType | undefined>(undefined);
+
+export const XplorersAIContextProvider: React.FC<{
+  children: React.ReactNode;
+}> = ({ children }) => {
+  const [value, setValue] = useState<string>("");
+  const [loading, setLoading] = useState<Boolean>(false);
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)